tests/test_plugin_flask.py

```diff
@@ -1,11 +1,14 @@
+from io import BytesIO
 from random import randint
 import pytest
 import json
 from flask import Flask, jsonify, request
+from werkzeug.datastructures import FileStorage
 
 from spectree import SpecTree, Response, Request
+from spectree.types import MultipartFormRequest
 
-from .common import Query, Resp, JSON, Headers, Cookies
+from .common import Query, Resp, JSON, Headers, Cookies, DemoModel
```

Comment:
Missed this. Why was it named `DemoModel`

Reply:
Nevermind. I think I got it.

Reply:
I think it's just an example schema.